LiteDB/Engine/Disk/Streams/AesStream.cs

```diff
@@ -52,6 +55,9 @@ public AesStream(string password, Stream stream)
             // start stream from zero position
             _stream.Position = 0;
 
+            var checkBuffer = bufferPool.Rent(32);
```

Comment:
``checkBuffer.Length`` will be used. Since the required 32 bytes is a minimum, the byte array returned could be larger.
Please adjust the code that uses the checkbuffer.

Reply:
which test should fail if the array's length isn't 32bytes?